app/lib/tax_return_status.rb

```diff
@@ -54,7 +54,7 @@ def message_templates
   PAUSED_STATUSES = [:file_not_filing, :file_hold]
```

Comment:
went looking in the code to remind myself what these `PAUSED_STATUSES` are - do you know what clients will see when they log into the portal if they're in `file_fraud_hold`?